GameDataParser/Parsers/MapParser.cs

```diff
@@ -2,53 +2,32 @@
 using System.Linq;
 using System.Text.RegularExpressions;
 using System.Xml;
-using GameDataParser.Crypto.Common;
 using GameDataParser.Files;
+using Maple2.File.Flat;
+using Maple2.File.Flat.maplestory2library;
+using Maple2.File.IO.Crypto.Common;
+using Maple2.File.Parser.Flat;
+using Maple2.File.Parser.MapXBlock;
 using Maple2Storage.Types;
 using Maple2Storage.Types.Metadata;
 
 namespace GameDataParser.Parsers
 {
     public class MapParser : Exporter<List<MapMetadata>>
     {
+        private List<MapMetadata> mapsList;
+        private Dictionary<int, string> mapNames;
```

Comment:
we use PascalCase and this file needs some formatting to remove empty white spaces